### DISCUSSION TITLE
How to add HEx Color Codes

### DISCUSSION THREAD

**Discussion by Nacholebraa** [EssentialsX/Essentials#4353] [Q&A] [answered]
I was wondering how individuals are able to add hex color codes to the system messages. Such as the following. 

Join Message | #28B463(+) #58D68D{USERNAME}
Quit Message | #CB4335(-) #EC7063{USERNAME}

**Comment by Octol1ttle** [comment, marked as answer]
Use `&#RRGGBB`
Your examples should look like this:
Join Message | &#28B463(+) &#58D68D{USERNAME}
Quit Message | &#CB4335(-) &#EC7063{USERNAME}

**Reply by Nacholebraa** [reply]
So, how would you do a gradient? Or would this feature not be supported?

**Reply by Nacholebraa** [reply]
Also, appreciate the help with it, It worked once adjusted.

**Reply by triagonal** [reply]
There is no inbuilt gradient support, but you can use a generator like https://rgb.birdflop.com/ to create one yourself. Note that this only works for static text, so you won't be able to dynamically apply it to every username using something like `{USERNAME`}.
